snippets/snippets.xml

```diff
@@ -0,0 +1,69 @@
+    <variable name="FileName" expression="" defaultValue="" alwaysStopAt="true" />
+    <variable name="MyComponent" expression="" defaultValue="" alwaysStopAt="true" />
+    <context>
+        <option name="DART" value="true" />
+    </context>
+</template>
+
+<template name="orAbsStless" value="import 'package:over_react/over_react.dart';&#10;&#10;// ignore: uri_has_not_been_generated&#10;part '$FileName$.over_react.g.dart';&#10;&#10;@AbstractProps()&#10;abstract class _$$$MyComponent$Props extends UiProps {}&#10;&#10;// AF-3369 This will be removed once the transition to Dart 2 is complete.&#10;// ignore: mixin_of_non_class, undefined_class&#10;abstract class $MyComponent$Props extends _$$$MyComponent$Props with _$$$MyComponent$PropsAccessorsMixin {&#10;  // ignore: undefined_identifier, undefined_class, const_initialized_with_non_constant_value&#10;  static const PropsMeta meta = $$metaFor$MyComponent$Props;&#10;}&#10;&#10;@AbstractComponent()&#10;abstract class $MyComponent$Component&lt;T extends $MyComponent$Props&gt; extends UiComponent&lt;T&gt; {&#10;  @override&#10;  Map getDefaultProps() =&gt; (newProps());&#10;&#10;  @override&#10;  render() {}&#10;}" description="Creates an abstract stateless over_react component" toReformat="false" toShortenFQNames="true">
+<variable name="FileName" expression="" defaultValue="" alwaysStopAt="true" />
+<variable name="MyComponent" expression="" defaultValue="" alwaysStopAt="true" />
```

Comment:
#nit It might be nice to give these default values so it's easier to see what you're filling in:
```
<variable name="FileName" expression="" defaultValue="&quot;foo&quot;" alwaysStopAt="true" />
<variable name="MyComponent" expression="" defaultValue="&quot;Foo&quot;" alwaysStopAt="true" />
```

![template](https://user-images.githubusercontent.com/1713967/50488537-3b040200-09c0-11e9-9a7f-6d2def071424.gif)